modules/features/discover/src/main/res/layout/row_podcast_large_list_with_podcast.xml

```diff
@@ -55,6 +55,8 @@
                 tools:text="The Daily"
                 style="@style/H20"
                 android:includeFontPadding="false"
+                android:maxLines="2"
+                android:ellipsize="end"
```

Comment:
This is a separate change but on the Discover tab if the new recommendation lists includes a long podcast title it can look like the following.

<img  width="300" src="https://github.com/user-attachments/assets/dd302b57-1046-4210-995d-b716410d77f6" />